src/app/Features/3Commas/API/api.ts

```diff
@@ -368,7 +368,6 @@ async function getAccountSummary(key:string , secret:string) {
   let api;
   if(key && secret) {
     api = threeCapi(config, key, secret)
```

Comment:
Removed the else statement here since it will mostly be the standard api. key/secret is only fed on the `check API keys` button in settings.